docs/config.json

```diff
@@ -274,6 +274,16 @@
         }
       ]
     },
+    {
+      "icon": "connect",
+      "title": "Use Teleport",
+      "entries": [
+        {
+          "title": "Using TSH",
+          "slug": "/use-teleport/tsh/"
+        }
+      ]
+    },
```

Comment:
I'm not sure if we still want to keep calling this entry "Using TSH" in the sidebar but so far I just moved stuff around without changing anything.